util/MovieLensParsing/src/RatingBuilder.java

```diff
@@ -0,0 +1,35 @@
+public class RatingBuilder {
+	private String userId;
+	private String movieId;
+	private String rating;
+	private String timestamp;
+	private boolean isUsed;
+
+	public Rating build() throws Exception {
+		if (isUsed)
+			throw new Exception("This rating builder has already been used");
```

Comment:
This is most likely not a terminal exception; it is a nice guard to have, but in most cases having explicit exception handling would be more annoying than not. Use a RuntimeException to not specify the throws clause and make it more readable.